### ISSUE TITLE
HTML output for large infrastructures and complex usecases 

### ISSUE THREAD

**Issue by sundowndev** [cloudskiff/driftctl#368]
TL;DR: create a new output to visualize scan result in a web page.

## Motivation

We usually see users struggling with the output of the scan in their terminal because it's too large and contain too much information. I was thinking about a new output that allow the user to visualize the output in their browser. Exactly the same way common code coverage tools work, we could generate an HTML file that contain a minimal web UI to display the scan result in a more human-readable way. I think the graphical way of viewing the scan result would be a pretty handy feature, and it could make the user better understand what driftctl is capable of.

Example with `go tool` : 

```
$ go tool cover -html=coverage.out
```

The coverage graphic will then look like this in the browser : 

![image](https://user-images.githubusercontent.com/16480203/112634901-73167b80-8e3b-11eb-9b4e-0fd4c8f8f617.png)

### Examples

```
$ driftctl scan --output html://result.html
```

The generated web page would then contain a list of managed, unmanaged, deleted resources, as well as differences that may exist. Since the web page is intended to be minimal, we have to focus more on UX, less on UI.

Here's my [first wireframe](https://wireframe.cc/XnRMox) :  

![image](https://user-images.githubusercontent.com/16480203/113727523-7cce9780-96f5-11eb-8b7d-290492dc240c.png)

## Implementation

I pretty familiar with Vuejs & Vuetify so I though about them in the first place. For example we can create a single file Vue app and use it to generate the output, without any Vue compilation. To achieve that we would embed the assets in the binary with the awesome [Go 1.16 feature](https://pkg.go.dev/embed). But we could also create a HTML template from scratch with no UI framework at all. We still have to discuss about that together.

**Comment by sundowndev** [comment]
For the record, here are a few screenshots of the version that got approved : 

![image](https://user-images.githubusercontent.com/16480203/122954973-30076e00-d380-11eb-9db2-cee6448e1d6d.png)

![image](https://user-images.githubusercontent.com/16480203/122955259-3e558a00-d380-11eb-9404-08dca74113e9.png)

![image](https://user-images.githubusercontent.com/16480203/122955306-49101f00-d380-11eb-81d0-706fe56db40f.png)

**Comment by sjourdan** [comment]
it's really cool! :rocket:
